docs/contributing/contributing.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Contributing to PCSX2"
+summary: "Getting started with contributing to PCSX2"
+draft: false
+toc: true
+sidebar_position: 1
+---
+
+So you want to contribute to PCSX2? Great
```

Comment:
Formality. At least put an exclamation mark at the end, but maybe consider something formal.